src/ui/generic/exception.tsx

```diff
@@ -62,8 +63,12 @@ const styles = StyleSheet.create({
   },
   buttonBlock: {
     marginTop: Spacing.LG,
-    flex: 0.8,
-    justifyContent: 'space-evenly',
+    justifyContent: BP<'space-between' | 'space-evenly'>({
+      small: 'space-between',
+      medium: 'space-evenly',
+      large: 'space-evenly',
+    }),
```

Comment:
On big devices the buttons are really spread out (e.g. iPhone 11). Maybe align them in the center with a fixed margin in between?